bin/steps/sqlite3

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/env bash
+
+set +e
+
+# shellcheck source=bin/utils
+source $BIN_DIR/utils
+
+SQLITE3_VERSION=$(apt-cache policy libsqlite3-dev | grep Candidate | sed 's/  Candidate: //')
+
+SQLITE3_VERSION_FILE="$BUILD_DIR/.heroku/python-sqlite3-version"
+if [ -f $SQLITE3_VERSION_FILE ]; then
+  INSTALLED_SQLITE3_VERSION=$(cat $SQLITE3_VERSION_FILE)
+fi
+
+# only install if the sqlite3 version has changed
+if [[ $INSTALLED_SQLITE3_VERSION != $SQLITE3_VERSION ]]; then
```

Comment:
I am not sure that this will ever be true, the cache is blank, so `$INSTALLED_SQLITE3_VERSION` is null, `apt-cache policy libsqlite3-dev | grep Candidate | sed 's/  Candidate: //'` will only return if `apt-get update` is run first, so I am not sure if this will work in the current context of the buildpack? If it is the case that `apt-cache policy libsqlite3-dev | grep Candidate | sed 's/  Candidate: //'` returns null then this condition will not be true unless the `$BUILD_DIR/.heroku/python-sqlite3-version` file exists, which it will not because it is only created at the end of this conditional block. Can anyone confirm this?